### PR TITLE
docs: improve contributing deprecation notice

### PR DESCRIPTION
**Pull request by Panquesito7**
Things added/changed:

- Improve contributing deprecation notice.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -503,4 +503,4 @@ We are running a Discord server. You can go here to talk, discuss, and more with
 | Visual Studio Code | https://code.visualstudio.com/                     | A code editor for editing code                     |
 | vscode.dev         | https://vscode.dev/                                | Visual Studio Code in the browser                  |
 | Iloveimg           | https://www.iloveimg.com/resize-image/resize-svg   | Resizing SVG's                                     |
-| svgviewer.dev      | https://www.svgviewer.dev/                         | View, save, and optimize SVGs                      |
\ No newline at end of file
+| svgviewer.dev      | https://www.svgviewer.dev/                         | View, save, and optimize SVGs                      |
```
Comment (Snailedlt) [inline CONTRIBUTING.md:506]: I don't see a change here. Could you point out what exactly changed on this line?

**Review by Snailedlt — CHANGES_REQUESTED** [review]
Looks good, but for some reason there seems to be an empty change at line 506. Any idea what that is?

File: CONTRIBUTING.md (hunk 1 above)
Comment (Panquesito7) [reply CONTRIBUTING.md:506]: I'm not sure about that, but I didn't make any changes there.

Comment (BenSouchet) [reply CONTRIBUTING.md:506]: it's maybe the line ending / line break, check https://stackoverflow.com/questions/1552749/difference-between-cr-lf-lf-and-cr-line-break-types

Comment (Snailedlt) [reply CONTRIBUTING.md:506]: I'll approve once this diff is fixed. Invisible changes always leave me with a bad taste in the mouth

**Review by Snailedlt — CHANGES_REQUESTED** [review]

**Comment by Panquesito7** [comment]
> I'll approve once this diff is fixed

I'm not sure that can be fixed. I could try but that's something that I didn't change.

**Comment by Panquesito7** [comment]
I found the issue: it seems there was a missing endline in the original file, and on this file, it was automatically added.
That's OK as some IDEs could have issues without an endline. There should be no issues with that. 🙂

**Review by Snailedlt — APPROVED** [review]

**Pull request merged**